src/popup/components/AliasMoreOptions.vue

```diff
@@ -177,9 +218,30 @@ export default {
       }
       this.loading = false;
     },
+
     toggleAliasPGP() {
       this.moreOptions.disable_pgp = !this.moreOptions.disable_pgp;
     },
+
+    findIndexOfMailbox(id, mailboxes) {
+      let index = -1;
+      for (const i in mailboxes) {
+        if (mailboxes[i].id === id) {
+          index = i;
+        }
+      }
+      return index;
+    },
+
+    toggleMailbox(mailbox) {
+      const i = this.findIndexOfMailbox(mailbox.id, this.moreOptions.mailboxes);
+      if (i === -1) {
+        this.moreOptions.mailboxes.push(mailbox);
+      } else {
+        this.moreOptions.mailboxes.splice(i, 1);
+      }
+      this.hasMailboxesChanges = true;
```

Comment:
`hasMailboxesChanges ` is true if user deselects and selects the same mailbox. I wonder if we can keep the original alias mailboxes and compare it with the `moreOptions.mailboxes` to know whether there's indeed a mailbox change?

Reply:
Yes, actually I thought about it and I came up with 3 solutions:
- check if `moreOptions.mailboxes === (original) alias.mailboxes` => doesn't work because it compares references
- `JSON.stringify(moreOptions.mailboxes) === JSON.stringify(alias.mailboxes)` => quite bad performance and won't work if the order of mailboxes is changed
- sort and compare only id => quite complex and bad for performance

so in the end, I decide not to do that

Reply:
implemented on c24a71d74e3d2d02404864d40b41403da32188e2